filament/backend/src/opengl/PlatformGLX.h

```diff
@@ -36,6 +38,7 @@ class PlatformGLX final : public backend::OpenGLPlatform {
     void terminate() noexcept override;
 
     SwapChain* createSwapChain(void* nativewindow, uint64_t& flags) noexcept override;
+    SwapChain* createSwapChain(uint32_t width, uint32_t height, uint64_t& flags) noexcept override;
```

Comment:
Use `final` for consistency with other platforms? (in fact, all of these methods could probably be made `final`)